src/pages/ResearcherProfile.js

```diff
@@ -263,6 +265,20 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     return isValid;
   };
 
+  isValidState(value) {
+    let isValid = false;
+    if (UnitedStatesOrEmpty.includes(this.state.profile.country.toLowerCase())) {
```

Comment:
This method takes a value to compare, so you can use that instead of looking at state.
```suggestion
    if (UnitedStatesOrEmpty.includes(value.toLowerCase())) {
```

Reply:
However, you'll want to make sure it's not nil before trying to do the include check.